adding docs change log (#1318)
temporalio/documentation: change-log/may-release-2022.md

---
slug: may-release-2022
title: May 23 2022
tags:
  - docs-change-log
date: 2022-05-23T00:00:00Z
---

> - `/docs` has been removed from the URLs of documentation pages.

Reviewer: ```suggestion
- We removed `/docs` from the URLs of documentation pages.
```
